nnoir-onnx/README.md

```diff
@@ -41,6 +41,8 @@ onnx2nnoir -o model.nnoir mnist/model.onnx
 * [MaxPool](https://github.com/onnx/onnx/blob/master/docs/Operators.md#MaxPool)
 * [Pad](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Pad)
     * `mode` must be `"constant"`
+* [PRelu](https://github.com/onnx/onnx/blob/master/docs/Operators.md#PRelu)
+    * `slope` must be `"constant"`
```

Comment:
We should use more restricted representation than "constant".  Not only "constant", but it must also be a singleton value.

Reply:
Thanks. I fixed.